src/psyclone/tests/psyir/transformations/arrayaccess2loop_trans_test.py

```diff
@@ -187,17 +182,17 @@ def test_apply_third_dim_expr(tmpdir):
         "  a(1,n,2*n+1) = b(1,n,2*n+1)\n")
     expected_result = (
         "  real, dimension(10,10,10) :: a\n  real, dimension(10,10,10) :: b\n"
-        "  integer :: n\n  integer :: jk\n\n"
-        "  do jk = 2 * n + 1, 2 * n + 1, 1\n"
-        "    a(1,n,jk) = b(1,n,jk)\n"
+        "  integer :: n\n  integer :: idx\n\n"
+        "  do idx = 2 * n + 1, 2 * n + 1, 1\n"
+        "    a(1,n,idx) = b(1,n,idx)\n"
         "  enddo\n\n")
     check_transformation(tmpdir, code, expected_result, index=2)
 
 
 def test_apply_fifth_dim_expr(tmpdir):
     '''Check that the expected code is produced for a 5D array where there
-    is no loop index information in the nemo api for the outermost
-    dimension so a new loop variable name is required.
+    is no loop index information for the outermost dimension so a new loo
```

Comment:
"loop"

Reply:
In fact, now that we're not looking up the name of the loop variable to use I think this test can be removed entirely?

Reply:
Agreed, removed the test.